tests/tests.rs

```diff
@@ -65,14 +65,206 @@ fn get_basic_cmt(file: &str) -> String {
 <!-- struct Foo§variant A -->
  Foo comment
  fn some_func(a: u32,
-              b: u32) {}
+              b: u32) {{}}
 <!-- mod Bar§macro test!§struct SuperFoo -->
  struct inside macro
 <!-- mod Bar§macro test!§macro sub_test!§struct FooFoo -->
  and another one!
-"#, file, "{}")
+"#, file)
 }
 
+const BASIC2 : &'static str = r#"
+use Bin;
+use Box;
+use Buildable;
+use Container;
+use Widget;
+use Window;
+use ffi;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::translate::*;
+
+glib_wrapper! {
+/// Dialog boxes are a convenient way to prompt the user for a small amount
```

Comment:
The indent is missing here. The test will still fail because it'll regenerate with an indent of four spaces.